### PR TITLE
Ido Improvements

### LINKED ISSUE #340: IDO Improvements
A.C:

[X] Register Investor is not decreasing by enough units
[X] InfoInvestor
[X] When a round expires an event should be generated
[X] Remove Operator Commission
[X] When is the voting decision taken
[X] Remove Show Interest (UI)
[X ] Fix AccountData issue (type)
[X ] Default of PDEX funding round creator
[X ] Governance config is wrong

### PR DESCRIPTION
**Pull request by mambisi**
Resolves #340

### CODE REVIEW

File: pallets/Polkadex_Ido/src/lib.rs [hunk 1]
```diff
@@ -401,6 +404,9 @@ decl_module! {
             };
             ensure!(token_a.ne(&token_b), <Error<T>>::TokenAEqTokenB);
 
+            let start_block = vote_end_block.clone().saturating_add(1_u128.saturated_into());
+            let close_round_block = vote_end_block.clone() + funding_period;
```
Comment (simonsso) [inline pallets/Polkadex_Ido/src/lib.rs:408]: saturating_add here too

File: pallets/Polkadex_Ido/src/lib.rs [hunk 2]
```diff
@@ -561,7 +534,7 @@ decl_module! {
             let current_block_no = <frame_system::Pallet<T>>::block_number();
             // Ensure investors can only show interest during the voting period of the round
             // Investors who shows interested are randomly selected (whitelisted)  to participate in the round at round start block
-            ensure!(current_block_no < funding_round.start_block, <Error<T>>::NotAllowed);
+            ensure!(current_block_no >= funding_round.start_block, <Error<T>>::NotAllowed);
```
Comment (simonsso) [inline pallets/Polkadex_Ido/src/lib.rs:537]: Should there not be two checks? and is `<` vs `>` correct?

File: pallets/Polkadex_Ido/src/test.rs [hunk 3]
```diff
@@ -167,86 +162,62 @@ fn test_whitelist_investor() {
...
-            PolkadexIdo::participate_in_round(Origin::signed(ALICE.clone()), round_id, balance),
-            Error::<Test>::NotWhiteListed
-        );
 
         assert_eq!(
             PolkadexIdo::register_round(
-                Origin::signed(ALICE.clone()),
+                Origin::signed(ALICE),
                 cid,
                 AssetId::CHAINSAFE(H160::from_low_u64_be(24)),
```
Comment (simonsso) [inline pallets/Polkadex_Ido/src/test.rs:176]: Did I not break this last week?

File: pallets/Polkadex_Ido/src/test.rs [hunk 4]
```diff
@@ -428,7 +402,9 @@ fn test_show_interest_in_round_randomized_participants() {
             );
         }
 
-        let funding_round: FundingRound<Test> = <InfoFundingRound<Test>>::get(round_id);
+
```
Comment (simonsso) [inline pallets/Polkadex_Ido/src/test.rs:405]:
```suggestion
```

File: pallets/Polkadex_Ido/src/test.rs [hunk 5]
```diff
@@ -655,8 +627,7 @@ fn create_hash_data(data: &u32) -> <mock::Test as frame_system::Config>::Hash {
 #[test]
 fn test_vote_for_round() {
     let balance: Balance = 100;
-    let open_block_number = 11;
-    let closing_block_number = 50;
+    let funding_period : BlockNumber = 10;
```
Comment (simonsso) [inline pallets/Polkadex_Ido/src/test.rs:630]: use other values in test too.

File: pallets/Polkadex_Ido/src/lib.rs (hunk 2 above)
Comment (mambisi) [reply pallets/Polkadex_Ido/src/lib.rs:537]: From the last discussions we are using the `show_interest`  also participates the the investor so, must called between on a block `funding_round.start_block` and `funding_round.close_round_block`

Comment (mambisi) [reply pallets/Polkadex_Ido/src/lib.rs:537]: From the last discussions we are using the `show_interest`  also participates the the investor so, must called between on a block `funding_round.start_block` and `funding_round.close_round_block`

File: pallets/Polkadex_Ido/src/test.rs (hunk 3 above)
Comment (mambisi) [reply pallets/Polkadex_Ido/src/test.rs:176]: I am branching from develop which is not updated to the lastest change from the main poldexprimitives repo

File: pallets/Polkadex_Ido/src/lib.rs (hunk 2 above)
Comment (simonsso) [reply pallets/Polkadex_Ido/src/lib.rs:537]: Yes so there should be two checks

**Review by simonsso — APPROVED** [review]

**Review by simonsso — APPROVED** [review]

**Pull request merged**